File: beginner/pagina12/3076/3076.cpp
#include <iostream>

using namespace std;

int main(){
    int ano;

    while(cin >> ano){
        if(ano <= 100){
            cout << 1 << endl;
        } else {

            int seculo = ano / 100;
            if(ano % 100 != 0){
                seculo++;
            }
            cout << seculo << endl;
        }
    }

    return 0;
}

File: beginner/__3000__/3076/3076.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner in = new Scanner(System.in);

        while(in.hasNextInt()){
            int ano = in.nextInt();

            if(ano <= 100){
                System.out.println(1);
            } else {

                int seculo = ano / 100;
                if(ano % 100 != 0){
                    seculo++;
                }

                System.out.println(seculo);
            }
        }
    }
}